### DISCUSSION TITLE
labs.lie not available even after installing from source

### DISCUSSION THREAD

**Discussion by nea98** [facebookresearch/theseus#617] [General]
I wanted to try labs package for [this](https://github.com/facebookresearch/theseus/issues/290#issuecomment-1538402616) application.
I cloned the repo and installed theseus from source. Still I am getting this error.
`ModuleNotFoundError: No module named 'theseus.labs.lie'`

**Comment by luisenp** [comment]
Hi @neanea04. The post you linked is old. `theseus.labs.lie` is now available under `torchlie` (`pip install torchlie`). See some examples [here](https://github.com/facebookresearch/theseus/tree/main/torchlie).
